docs/src/pages/[platform]/components/authenticator/configuration/index.page.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Configuration
+metaTitle: Configuration
+description: How to setup and configure your Authenticator component.
+supportedFrameworks: react|angular|vue
+---
+
+import { Fragment } from '@/components/Fragment';
+import { Alert, Tabs, TabItem } from '@aws-amplify/ui-react';
+
+### Zero Configuration
```

Comment:
Bump all these headings to an h2 since this section is its own page now.

Reply:
OK done #3fa06580eb09862a439986cc89893f9f241d7ddf